message/message1_1/transfer_message.go

```diff
@@ -4,31 +4,32 @@ import (
 	"bytes"
 	"io"
 
-	datatransfer "github.com/filecoin-project/go-data-transfer"
 	"github.com/ipld/go-ipld-prime/codec/dagcbor"
 	"github.com/ipld/go-ipld-prime/datamodel"
 	basicnode "github.com/ipld/go-ipld-prime/node/basic"
+
+	datatransfer "github.com/filecoin-project/go-data-transfer"
 )
 
-//go:generate cbor-gen-for --map-encoding transferMessage1_1
+//go:generate cbor-gen-for --map-encoding TransferMessage1_1
 
 // transferMessage1_1 is the transfer message for the 1.1 Data Transfer Protocol.
-type transferMessage1_1 struct {
+type TransferMessage1_1 struct {
```

Comment:
Question: if these are all the old types, why are we renaming them?

Reply:
it just makes the change set larger

Reply:
also, since they exist ONLY for a compatibility test, I wonder if the directory structure should be simply

`message1_1` - Prime nodes
`message1_1/cbor_gen_legacy_test` (or something like that) - this directory?

Reply:
It was something about getting the tests to pass, which are in a `message1_1test` package and are circular if you remove the `test`. I don't recall why I needed to do this but it ended up being the easiest path to solving it ... In my next PR I've just removed this directory, it became too hard to keep intact because it pulls in the FakeDType stuff which needs to change too.